src/components/autocomplete/js/autocompleteController.js

```diff
@@ -618,7 +618,7 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
     // Set the loading to true so we don't see flashes of content.
     // The flashing will only occour when an async request is running.
     // So the loading process will stop when the results had been retrieved.
-    setLoading(true);
+    if ($scope.searchText) setLoading(true);
```

Comment:
I think, since the clear button is hidden, when the search text is empty, the autocomplete should not call `clearValue()` when pressing <kbd>ESC</kdb> as well.